packages/api-client/package.json

```diff
@@ -28,13 +28,11 @@
     "url": "https://github.com/GlobalFishingWatch/frontend/issues"
   },
   "dependencies": {
-    "@globalfishingwatch/api-types": "^4.0.0",
     "file-saver": "^2.0.2"
   },
-  "peerDependencies": {
-    "@globalfishingwatch/pbf": "^1.0.10"
-  },
   "devDependencies": {
+    "@globalfishingwatch/api-types": "^4.0.0",
+    "@globalfishingwatch/pbf": "^1.0.10",
```

Comment:
This is not perfect but we need to have the `@globalfishingwatch` dependencies in both `devDependencies` and `peerDependencies` to:
- ensure built is done in order (dev)
- not include the dependency in the bundle (peer)